packages/wxt/src/core/resolve-config.ts

```diff
@@ -426,17 +424,21 @@ async function getUnimportEslintOptions(
  * Returns the path to `node_modules/wxt`.
  */
 function resolveWxtModuleDir() {
-  // TODO: Use this once we're fully running in ESM, see https://github.com/wxt-dev/wxt/issues/277
-  // const url = import.meta.resolve('wxt', import.meta.url);
-  // resolve() returns the "wxt/dist/index.mjs" file, not the package's root
-  // directory, which we want to return from this function.
-  // return path.resolve(fileURLToPath(url), '../..');
-
-  const nodeRequire = globalThis.require ?? createRequire(import.meta.url);
-
-  // resolve() returns the "wxt/dist/index.mjs" file, not the package's root
+  // TODO: Drop the __filename expression once we're fully running in ESM
+  // (see https://github.com/wxt-dev/wxt/issues/277)
+  const importer =
+    typeof __filename === 'string'
+      ? pathToFileURL(__filename).href
+      : import.meta.url;
```

Comment:
Not 100% sure the `__filename` check is necessary, tbh. This was added defensively, in case the Node.js version doesn't have `import.meta.url`.

Reply:
We definitely need to keep it in. I don't want to switch to using `import.meta.url` when older versions of node 18 and node 20 don't support it yet. v0.20 will migrate us fully to ESM, at which point I'm happy removing the `__filename` check. But that needs to come with a breaking change, it can't be in a patch request.

Reply:
Or maybe I'm misremembering... It looks like I was using it before? But after checking for the global `require` function... Hmm, not sure. I always have to look things up whenever it comes to ESM/CJS interop